site/templates/application/google.jet

```diff
@@ -1,24 +1,32 @@
-{{block googleGa4Script(analyticsId=config("google_analytics_id"))}}
-  {{if hasPrefix(analyticsId,"G")}}
-  <!-- GA4 script if G tracking code is provided -->
-    <script async src="https://www.googletagmanager.com/gtag/js?id={{analyticsId}}"></script>
-  {{end}}
-{{end}}
-
 {{block googleScripts(tagManagerId=config("google_tag_manager_id"), analyticsId=config("google_analytics_id"))}}
   <!-- Google integration scripts -->
   <script>
     function loadGoogleTagManager() {
-      (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
-      new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
-      j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
-      'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
-      })(window,document,'script','dataLayer','{{ tagManagerId }}')
+      (
```

Comment:
ah i de-uglified this to make sense of it. do i need to revert it?

Reply:
Yeah, revert back to the ugly since it's what we've copied from Googles snippets.